plugins/export.js

```diff
@@ -63,7 +64,7 @@ window.Components = {
     ChannelMembersModal,
     Avatar,
     imageURLForUser,
-    BotBadge,
+    BotBadge: BotTag,
```

Comment:
But the change here is minimal. IMO this can stay as is, but we should have our eyes open for any upcoming bug tickets.

The API for the component did not change, so it can just be used as-is